### ISSUE TITLE
wall o compile node_module/@types errors when you run tests

### ISSUE THREAD

**Issue by tomwayson** [Esri/esri-loader#290]
running `yarn test` results in a wall o compile errors for `node_modules/@types` packages:

<img width="959" alt="image" src="https://user-images.githubusercontent.com/662944/160666489-063c1d91-6aa2-4961-8b8b-48999895e2d6.png">

Tests still pass.

I resolved these compile errors for `yarn build` in 766081f96d03b9735cb2f7147e78374bf78075e0, but when I tried to add that to the [karma-typescript config](https://www.npmjs.com/package/karma-typescript#Advanced-configuration) like:

```js
karmaTypescriptConfig: {
    compilerOptions: {
      types: []
    }
}
```

The tests wouldn't even run. May need to update karma-typescript version.

**Comment by andygup** [comment]
@tomwayson I had to pull `master` to merge your fix. And, then I updated `karma-typescript` and `typescript` to `latest`. Now everything is running correctly on my local machine. 

Here's my package.json:

```
  "devDependencies": {
    "@types/jasmine": "^2.8.11",
    "concurrently": "^3.4.0",
    "jasmine-core": "^2.8.0",
    "karma": "^6.3.16",
    "karma-chrome-launcher": "^2.0.0",
    "karma-coverage": "^1.1.2",
    "karma-firefox-launcher": "^1.1.0",
    "karma-jasmine": "^1.1.0",
    "karma-mocha-reporter": "^2.2.3",
    "karma-typescript": "^5.5.3", <== this is updated
    "mkdirp": "^0.5.1",
    "onchange": "^3.2.1",
    "rimraf": "^2.6.2",
    "rollup": "^0.41.6",
    "rollup-plugin-uglify": "^2.0.1",
    "tslint": "^5.7.0",
    "typescript": "^4.6.3". <== this is updated
  }

```

<img width="698" alt="image" src="https://user-images.githubusercontent.com/510440/160667592-fe00c8c3-d87c-4630-97e8-56dd8f596161.png">

**Comment by tomwayson** [comment]
resolved in #292
